aws-samples/amazon-chime-sdk
# Setup Sam Deploy for moderated-chat-and-sentiment-analysis demo and real-time-collaboration demo

**Issue #:**

**Description of changes:**
1. Set up Sam Deploy for `moderated-chat-and-sentiment-analysis` demo.
2. Set up Sam Deploy for `real-time-collaboration` demo. 
3. Remove GeoRestriction on the CloudFront endpoint of `chat` demo.

**Testing**

1. How did you test these changes? 

- Run the `npm run deploy` command following new README to deploy both `moderated-chat-and-sentiment-analysis` and `real-time-collaboration` demos using the sam deploy method.
- Deploy both `moderated-chat-and-sentiment-analysis` and `real-time-collaboration` demos using the original CloudFormation method and ensure nothing break.

5. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it? Yes, they can be tested using `moderated-chat-and-sentiment-analysis`, `real-time-collaboration` and `chat` demos.
6. If applicable, have you run `npm run build` successfully locally to fix all warnings and errors? Yes

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
